### PR TITLE
Update the issue template to guide on bug trackers

### PR DESCRIPTION
**Pull request by danjenkins**
**Description**

Add urls to bug trackers for browsers/native libwebrtc sdk

**Purpose**

There are too many bugs in this repo related to browser or SDK issues... lets try and stop people making them in the first place.

Not sure if I got all the right URLs or my wording is good enough though

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Please read first!
 Please use [discuss-webrtc](https://groups.google.com/forum/#!forum/discuss-webrtc) for general technical discussions and questions.
+If you have found an issue/bug with the native `libwebrtc` SDK please create an issue at [bugs.webrtc.org](https://bugs.chromium.org/p/webrtc/issues/list)
+If you have found an issue/bug with one of the browser's behaviours around WebRTC please create an issue in the relevant issue tracker
+  - Chromium (Includes Chrome/Brave/Opera/Edge) - [bugs.webrtc.org](https://bugs.chromium.org/p/webrtc/issues/list)
```
Comment (alvestrand) [inline .github/ISSUE_TEMPLATE.md]: please use crbug.com in the text form (can also use that in the URL form).

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # Please read first!
 Please use [discuss-webrtc](https://groups.google.com/forum/#!forum/discuss-webrtc) for general technical discussions and questions.
+If you have found an issue/bug with the native `libwebrtc` SDK please create an issue at [bugs.webrtc.org](https://bugs.chromium.org/p/webrtc/issues/list)
```
Comment (alvestrand) [inline .github/ISSUE_TEMPLATE.md]: The canonical link is https://bugs.webrtc.org/ - which redirects to this URL, but we might as well give the canonical one.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+If you have found an issue/bug with one of the browser's behaviours around WebRTC please create an issue in the relevant issue tracker
+  - Chromium (Includes Chrome/Brave/Opera/Edge) - [bugs.webrtc.org](https://bugs.chromium.org/p/webrtc/issues/list)
+  - Firefox - [bugzilla.mozilla.org](https://bugzilla.mozilla.org/buglist.cgi?product=Core&component=WebRTC&resolution=---)
+  - Webkit (Includes Safari on MacOS, iOS and iPadOS) - [webkit.org/reporting-bugs](https://webkit.org/reporting-bugs/)
 
+- [ ] I understand issues created here are _only_ relevant to the samples in this repo - not browser or SDK bugs
```
Comment (alvestrand) [inline .github/ISSUE_TEMPLATE.md]: English nit: "I understand that issues..."

**Review by alvestrand — COMMENTED** [review]
Nits only. I think it is  a good addition.

**Review by alvestrand — APPROVED** [review]

**Review by fippo — APPROVED** [review]

**Comment by fippo** [comment]
maybe just link to https://webrtc.googlesource.com/src/+/refs/heads/main/docs/bug-reporting.md?
But the checkbox is important

**Comment by fippo** [comment]
squashed, thank you @danjenkins

**Pull request merged**